client/state/stats/lists/utils.js

```diff
@@ -238,6 +251,55 @@ export const normalizers = {
 	},
 
 	/**
+	 * Returns a normalized statsTopAuthors array, ready for use in stats-module
+	 *
+	 * @param  {Object} data   Stats data
+	 * @param  {Object} query  Stats query
+	 * @param  {Int}    siteId Site ID
+	 * @param  {Object} site   Site Object
+	 * @return {Array}       Normalized stats data
+	 */
+	statsTopAuthors( data, query = {}, siteId, site ) {
+		if ( ! data || ! query.period || ! query.date ) {
+			return [];
+		}
+		const { startOf } = rangeOfPeriod( query.period, query.date );
+		const authorsData = get( data, [ 'days', startOf, 'authors' ], [] );
+
+		return authorsData.map( ( item ) => {
+			const record = {
+				label: item.name,
+				iconClassName: 'avatar-user',
+				icon: parseAvatar( item.avatar ),
+				children: null,
+				value: item.views,
+				className: 'module-content-list-item-large',
+				actions: [ {
+					type: 'follow',
+					data: item.follow_data ? item.follow_data.params : false
+				} ]
+			};
+
+			if ( item.posts && item.posts.length > 0 ) {
+				record.children = item.posts.map( ( child ) => {
+					return {
+						label: child.title,
+						value: child.views,
+						page: site ? '/stats/post/' + child.id + '/' + site.slug : null,
```

Comment:
@timmyc I just saw that you used another approach for statTopPosts. You add the "domain" to the query object. Do you think we should keep it or should we use `site.slug`. I'll update the PR to use the same technique for all normalizers after the discussion ;)